typescript/src/offerings.ts

```diff
@@ -107,6 +107,10 @@ export interface PurchasesStoreProduct {
      * Product category.
      */
     readonly productCategory: PRODUCT_CATEGORY | null;
+    /**
+     * The specific type of subscription or one time purchase this product represents
```

Comment:
Can you add a warning to indicate that this isn't available for StoreKit 1 products? https://github.com/RevenueCat/purchases-ios/blob/4.26.1/Sources/Purchasing/StoreKitAbstractions/StoreProduct.swift#L111

Reply:
Did this in #515